src/constants.h

```diff
@@ -181,7 +181,7 @@ CONSTEXPR(uint32_t, captivePortalRequestTimeoutMsec, 10000, 4000, 0)
 CONSTEXPR(uint32_t, statusIconAnimationMsec, 200, 200, 0)
 
 // How often glean pings are sent
-CONSTEXPR(uint32_t, gleanTimeoutMsec, 1200000, 4000, 0)
+CONSTEXPR(uint32_t, gleanTimeoutMsec, 1200000, 1200000, 0)
```

Comment:
Because it makes it easier for testing if we are not sending one ping every four seconds. 

As you saw, I added a button to the dev menu for sending pings on demand. If we are also sending a ping every 4s that button doesn't help at all. IMO, it's easier to test sending pings on demand instead of sending them every four seconds, because we can have better control of what will be in the ping.

Reply:
That's definitely compelling, thanks for updating this.